tests/test_evolve/test_likelihood_function.py

```diff
@@ -1352,6 +1352,23 @@ def test_to_rich_dict(self):
             lf.set_alignment(_aln)
             _ = lf.to_rich_dict()
 
+        # tests multiple alignments
+        half = len(self.data) // 2
+        aln1 = self.data[:half]
+        aln2 = self.data[half:]
+        loci_names = ["1st-half", "2nd-half"]
+        loci = [aln1, aln2]
+        tree = make_tree(tip_names=self.data.names)
+        model = get_model("HKY85")
+        lf = model.make_likelihood_function(tree, loci=loci_names)
+        lf.set_alignment(loci)
+        for i, loci_name in enumerate(loci_names):
+            d = lf.to_rich_dict()
+            alignment = d["alignment"]
+            motif_probs = d["motif_probs"]
+            self.assertEqual(alignment[loci_name], loci[i].to_rich_dict())
+            self.assertEqual(motif_probs[loci_name], loci[i].get_motif_probs())
+
```

Comment:
add a test for when there's a single alignment.

In the new test, line 1363 become
```python
lf = model.make_likelihood_function(tree)
```
and line 1364 becomes
```python
lf.set_alignment(aln1)
```
There's no loop, and the result of `d["alignment"]` should equal `aln1.to_rich_dict()`